worlds/bumpstik/test/TestLogic.py

```diff
@@ -13,25 +13,27 @@ def testLogic(self):
 
                 if bb_count < 4:
                     self.assertFalse(self.can_reach_location(f"Treasure Bumper {x + 1}"))
+                    # Can't reach Treasure Bumper 9 check until level 2 is unlocked, etc.
+                    # But we don't have enough Treasure Bumpers to reach this check anyway??
                 elif bb_count == 4:
                     bb_count += 1
+                    # Level 4 has two new Bonus Booster checks; need to check both
 
-                for y in range(self.count("Booster Bumper"), bb_count):
-                    self.assertTrue(self.can_reach_location(f"Bonus Booster {y + 1}"),
-                                    f"BB {y + 1} check not reachable with {self.count('Booster Bumper')} BBs")
-                    if y < 4:
-                        self.assertFalse(self.can_reach_location(f"Bonus Booster {y + 2}"),
-                                         f"BB {y + 2} check reachable with {self.count('Treasure Bumper')} TBs")
-                    self.collect(self.get_item_by_name("Booster Bumper"))
+                for y in range(self.count("Booster Bumper"), bb_count + 1):
```

Comment:
Same suggestion, to replace the arbitrary name "y" for a proper name

Reply:
Done.